.github/workflows/pythonpackage.yml

```diff
@@ -0,0 +1,26 @@
+name: Upload Python Package
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: '3.x'
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install setuptools wheel twine
+    - name: Build and publish
+      env:
+        TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
+        TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
+      run: |
+        python setup.py sdist bdist_wheel
+        twine upload dist/*
```

Comment:
Just be careful as our pipeline already publishes to PyPi. This might raise a "Version already exists" error when publishing it with Twine.

Reply:
I understand that as it is only in the creation of the release, it has to give error. The release should only be done when we have the correct version in the master.

Reply:
My point was that Azure Devops might already be doing this, pretty much like Github, isn't it? Hence there might be a conflict with two pipelines trying to publish the same artifact to pypi